dclab/features/qpi/dry_mass.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+
+
+def get_dry_mass(masked_phase, wavelength, pixel_size,
+                 alpha_cell_type=0.19):
+    """
+    Calculate the dry mass from the phase image.
+
+    Parameters
+    ----------
+    masked_phase : np.ndarray
+        Phase image * event mask. Can be an image stack.
```

Comment:
Hey @paulmueller, I've been allowing image stacks to be used for calculation of features. Let me know if this isn't wanted in dclab!